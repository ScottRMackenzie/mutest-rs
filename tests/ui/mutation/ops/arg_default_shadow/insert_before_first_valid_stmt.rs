//@ print-mutations
//@ build
//@ stdout
//@ stderr: empty
//@ mutation-operators: arg_default_shadow

#![feature(decl_macro)]

macro dummy() {}

macro m($a:ident) {
    println!("m: {}", $a);
}

fn f(a: usize) {
    dummy!();
    m!(a);
}

#[test]
fn test() {
    f(1);
}
