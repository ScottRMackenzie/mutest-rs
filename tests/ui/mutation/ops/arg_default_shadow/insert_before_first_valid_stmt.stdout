[arg_default_shadow] ignore `a` argument by shadowing it with `Default::default()` in f at tests/ui/mutation/ops/arg_default_shadow/insert_before_first_valid_stmt.rs:15:6: 15:14
  <-(0)- test

1 mutations; 1 safe; 0 unsafe (0 tainted)
